lib/jsdom/living/index.js

```diff
@@ -42,6 +42,7 @@ exports.WheelEvent = require("./generated/WheelEvent").interface;
 exports.EventTarget = require("./generated/EventTarget").interface;
 
 exports.BarProp = require("./generated/BarProp").interface;
+exports.External = require("./generated/External").interface;
```

Comment:
So actually `External` is `[NoInterfaceObject]` so we should not do this. I guess we should try to make use of the `exposed` thingy some day.

Reply:
> I guess we should try to make use of the `exposed` thingy some day.

Already have a WIP :)